modules/preview-react/segmented-control/stories/SegmentedControl.stories.mdx

```diff
@@ -23,10 +23,9 @@ import {
 
 Segmented Control is a
 [compound component](/getting-started/for-developers/resources/compound-components/) that represents
-a linear group of two, three or four buttons allowing the selection of a specific value. This is a
-controlled component.
+a linear group of multiple buttons allowing the selection of a specific value.
```

Comment:
```suggestion
a linear group of multiple buttons allowing the selection of a specific value. We strongly encourage to use no more than 4 buttons.
```

Reply:
Got it, I was wondering if the inclusion of that limit was intentional.

Reply:
FYI: I moved this text to the `Basic Example` section. The paragraph at the top of the page (underneath the H1) is automatically stripped out by the Canvas Site so this guidance won't be visible on the Site unless we move it further down.